addon/components/power-select.ts

```diff
@@ -287,6 +287,24 @@ export default class PowerSelect extends Component<PowerSelectArgs> {
     }
   }
 
+  @action
+  handleMousedown() : void {
+    const trigger = document.getElementById(this.args.triggerId);
+    trigger?.setAttribute('mousedown', 'true');
+  }
+
+  @action
+  _open(event: FocusEvent): void {
```

Comment:
This allows the input to be focused as soon as the user tabs to it. We add the mousedown attribute in the handleMousedown action to prevent this action from closing blurring the input, after the user clicks on the trigger (clicking the trigger already focuses the input, so toggling the input again will make it blur and disapppear).